contracts/modules/common/LimitManager.sol

```diff
@@ -230,12 +145,22 @@ abstract contract LimitManager is BaseModule {
     * @param _pending The value of the pending parameter
     * @param _changeAfter The value of the changeAfter parameter
     */
-    function currentLimit(uint128 _current, uint128 _pending, uint64 _changeAfter) internal view returns (uint128) {
+    function currentLimit(uint256 _current, uint256 _pending, uint256 _changeAfter) internal view returns (uint256) {
         // solium-disable-next-line security/no-block-members
         if (_changeAfter > 0 && _changeAfter < now) {
             return _pending;
         }
         return _current;
     }
 
+    function safe128(uint256 _num) internal pure returns (uint128) {
+        require(_num < 2**128, "LM: more then 128 bits");
+        return uint128(_num);
+    }
+
+    function safe64(uint256 _num) internal pure returns (uint64) {
+        require(_num < 2**64, "LM: more then 64 bits");
+        return uint64(_num);
+    }
+
```

Comment:
This should probably live in the `ArgentSafeMath` library.

Reply:
The problem is that `ArgentSafeMath` is under `/module` while these methods are only used in `/infrastructure/storage`. 

If we want to do some reshuffling we should do it in a separate PR so I'll these 2 methods there for now.

Reply:
I've left them in the new `LimitUtils` library for now as they are only used in the context of the daily limit.